docs/feature-flagging/experiment-assignment.md

```diff
@@ -23,3 +23,51 @@ Click "Create Experiment" on the experiment allocation to create an experiment.
 Once assignments have begun, they will need to be logged to your data warehouse to be analyzed by Eppo. All SDKs take in a callback function that is called with the assignment parameters (`subject_id`, `experiment_id`, `timestamp`, etc.) passed in when assignments are made. That function is completely managed by you - the only requirement is that the assignment data eventually makes it to your warehouse. 
 
 Congratulations on setting up your first Eppo randomized experiment! Refer to our [analysis setup guide](/experiment-quickstart) to connect Eppo to your data warehouse and easily analyze the results.
+
+## Running no/low code experiments in Eppo
```

Comment:
The new material is great but it feels out of place, at least how it is currently organized. Creating multiple experiment allocations, reordering, archiving all could belong here. But the no/low-code angle and JSON types feels like it belongs elsewhere, either in the "Dynamic Configuration" page or as a stand-alone tutorial.

Reply:
thank you for the thorough review @giorgiomartini0 -- I'll review this @lukasgoetzweiss or @emetelka and make those updates

Reply:
Updated layout to have the Multiple Allocations in Experiment Assignment and the No Code and JSON flag section in Dynamic Config